### DISCUSSION TITLE
Dynamic albums based on metadata

### DISCUSSION THREAD

**Discussion by PilotJ357** [immich-app/immich#6552] [General]
I think a really helpful function would be the ability to dynamically create albums based on metadata within files. I've provided a few examples below:

- All photos from X camera, focal length, film stock (NegativeLabPro allows addition of film based metadata to files)
- All photos with a Lightroom rating of 3+
- All photos from the state of Hawaii (Geofencing, zones, or states..this could be more challenging to implement?)
- All photos with specific keywords

This could be configured by using the existing search functions (With some expansion required for examples above? Unless I just am not well versed at searching) as the lookup to allow for automatic additions for future photos.

**Comment by bo0tzz** [comment]
Duplicate of #1673
